### PR TITLE
Remove inline Javascript for administrator/components/com_newsfeeds/field/modal/newsfeedsfield

### PR DESCRIPTION
**Pull request by astridx**
Pull Request for Issue # .

### Summary of Changes
Remove Inline JavaScript of administrator/components/com_newsfeeds/field/modal/newsfeedsfield

### Testing Instructions
You can test in com_menu. You can create a menu item with a single newsfeed and select a newsfeed.

### CODE REVIEW

File: administrator/components/com_newsfeeds/Field/Modal/NewsfeedField.php [hunk 1]
```diff
@@ -42,6 +43,8 @@ protected function getInput()
 		$allowClear  = ((string) $this->element['clear'] != 'false');
 		$allowSelect = ((string) $this->element['select'] != 'false');
 
+		$doc = \JFactory::getDocument();
```
Comment (C-Lodder) [inline administrator/components/com_newsfeeds/Field/Modal/NewsfeedField.php]: Please remove this line

File: administrator/components/com_newsfeeds/Field/Modal/NewsfeedField.php [hunk 2]
```diff
@@ -67,12 +70,8 @@ protected function getInput()
 
 			if (!isset($scriptSelect[$this->id]))
 			{
-				Factory::getDocument()->addScriptDeclaration("
-				function jSelectNewsfeed_" . $this->id . "(id, title, object) {
-					window.processModalSelect('Newsfeed', '" . $this->id . "', id, title, '', object);
-				}
-				"
-				);
+				$doc->addScriptOptions('newsfeed_id', $this->id);
```
Comment (C-Lodder) [inline administrator/components/com_newsfeeds/Field/Modal/NewsfeedField.php]: and replace `$doc->` with `Factory::getDocument()->`

**Comment by dneukirchen** [comment]
pr for #55

**Comment by C-Lodder** [comment]
@astridx would you mind fixing the conflicts please?

**Comment by astridx** [comment]
This three files are not changed in this PR. They came in by mistake. Is there an easy way to exclude these files?

**Comment by C-Lodder** [comment]
I'm not sure how to revert a single file in Git, but I suppose you can just copy and paste the code from the file in the repo, then commit that.

**Comment by astridx** [comment]
Conflicts fixed.

**Review by dneukirchen — CHANGES_REQUESTED** [review]
Please revert changes to media action plugins

**Comment by astridx** [comment]
See: https://github.com/joomla-projects/joomla-es6/pull/106

**Pull request closed without merging**